.github/workflows/update.yml

```diff
@@ -38,3 +38,15 @@ jobs:
           branch: 'update-octokit-webhooks'
           author: 'github-actions <41898282+github-actions[bot]@users.noreply.github.com>'
           commit-message: 'feat: update `@octokit/webhooks-definitions`'
+      - name: trigger ci checks
+        uses: octokit/request-action@v2.x
+        id: workflow_dispatch
```

Comment:
@gr2m I'm assuming this property isn't important? the document doesn't indicate that it's got to match something.

Reply:
you only need `id` if you want to access the response in successive steps. You can safely remove it in this case

```suggestion
```